protractor.headless.conf.js

```diff
@@ -0,0 +1,56 @@
+﻿'use strict';
+
+var paths = require('./paths.config.js');
```

Comment:
eslint

Reply:
Der er en jasmine del under onPrepare som eslint ikke er glad for. Men denne er protractor tilgengæld meget glad for.